par-itérations.md

```diff
@@ -0,0 +1,60 @@
+# Gestion de projet agile - des itérations
+
+Une des composantes majeurs de dtc est d'accompagner les organisations avec/pour lesquelles nous travaillons. Cette idée d'accompagnement n'est pas seulement un positionnement dogmatique, mais une réponse très concrète à la complexité inherénte à faire des produits interactifs.
+
+Dans cette page, nous expliquons une partie importante de notre façon de travailler : l'approche par itération avec rendus réguliers.
+
+
+## La relation client-prestataire
+
+On peut schématiser "l'imaginaire client" classique avec le schéma suivant :
+
+![départ, flèche directe, arrivée](./images/itérations/fantasme.png)
+
+On peut découper 3 éléments dans ce schéma : 
+- le client sait où il en est
+- le client sait où il veut aller
+- le prestataire va prendre le chemin le plus rapide pour y arriver (la ligne droite)
+
+Malheureusement, la réalité n'est jamais aussi simple. D'après nos expériences professionelles, respectives, nous pouvons battre en brêche les 3 pré-supposés : 
+
+Même si c'est plutôt rare, il arrive le client ne connait pas bien son environnement, ne connait pas bien ses outils et/ou les personnes autour de lui/elle. Il/elle a déjà la réponse a son besoin sous la main, mais ne le voit pas. Pour la suite de cette page, nous présupposerons quand même que le client sait d'où il part.
+
+Par contre, il est très fréquent que le client ne sache pas ce qu'il veut. Il/elle l'exprimera avec des mots vagues ou insuffisamment précis pour commencer à construire avec confiance. Il est aussi extrêmement fréquent que le besoin du client évolue au cours du temps. Les priorités changent parfois. Ou, au contact de premiers rendus, le client se rend compte que la direction n'est pas la bonne. Ces facteurs sont inévitables. 
```

Comment:
J'emphaserais bien sur 

> Ces facteurs sont inévitables 

C'est clairement inévitable, c'est hyper important et c'est le rôle de l'accompagnant (pas celui du presta) que de faire accepter l'inévitable et de travailler avec plutôt que de travailler contre.